internal/admission/validator_test.go

```diff
@@ -384,4 +402,157 @@ func TestKongHTTPValidator_ValidateConsumer(t *testing.T) {
 	})
 }
 
+type fakeConsumerGroupSvc struct {
+	kong.AbstractConsumerGroupService
+	err error
+}
+
+func (f fakeConsumerGroupSvc) List(context.Context, *kong.ListOpt) ([]*kong.ConsumerGroup, *kong.ListOpt, error) {
+	if f.err != nil {
+		return []*kong.ConsumerGroup{}, &kong.ListOpt{}, f.err
+	}
+	return []*kong.ConsumerGroup{}, &kong.ListOpt{}, nil
+}
+
+type fakeInfoSvc struct {
+	kong.AbstractInfoService
+	version string
+}
+
+func (f fakeInfoSvc) Get(context.Context) (*kong.Info, error) {
+	if f.version != "" {
+		return &kong.Info{Version: f.version}, nil
+	}
+	return nil, kong.NewAPIError(http.StatusInternalServerError, "bogus fake info")
+}
+
+func TestKongHTTPValidator_ValidateConsumerGroup(t *testing.T) {
+	store, _ := store.NewFakeStore(store.FakeObjects{})
```

Comment:
Why don't we check the error here?

Reply:
Copied from another adjacent test (plugin validation) without much thought.

It looks like we don't really need it--we use it because the validator's supposed to have one, but it doesn't actually get used in CG validation, and could be omitted.